scripts/spm_test_schemes/FirebaseCoreInternalTests.xcscheme

```diff
@@ -1,26 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1300"
+   LastUpgradeVersion = "1320"
    version = "1.3">
    <BuildAction
       parallelizeBuildables = "YES"
       buildImplicitDependencies = "YES">
-      <BuildActionEntries>
```

Comment:
Keep it buildable

Reply:
Good catch! Done in [bb32a8b](https://github.com/firebase/firebase-ios-sdk/pull/9482/commits/bb32a8b550fcf3810295691b82941392d3278e40)